lessons/02-javascript/README.md

```diff
@@ -331,10 +338,10 @@ $ node functions.js
 This is totally legal. It might help to break down the expression we just printed:
 
 ```node
-func1()(); // first, func1 is called
+func1()(); // first, func1 is called and we say we will call its output
```

Comment:
I'd throw in the word 'immediately' as well, if only as a subtle hint toward IIFE's.
https://en.wikipedia.org/wiki/Immediately-invoked_function_expression